src/createRollupPrepareTransactionRequest.ts

```diff
@@ -72,6 +73,25 @@ export async function createRollupPrepareTransactionRequest<TChain extends Chain
     }
   }
 
+  const arbOSVersion = chainConfig.arbitrum.InitialArbOSVersion;
+  const wasmModuleRoot = params.config.wasmModuleRoot;
+
+  if (arbOSVersion === 30 || arbOSVersion === 31) {
```

Comment:
Both ArbOS 30 and ArbOS 31 shouldn't be used, so we explicitly throw in case they are used